custom-checks/src/main/java/io/opentelemetry/javaagent/customchecks/InternalJavadoc.java

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
If I were adding this to the SDK repo (which I do want to do, this looks sweet), I would add it to `buildSrc` and just keep it with our conventions (they are conventions). Can we do that here too?

Reply:
I tried but was unable to get that to work, see commit history, the problem I had was that errorprone doesn't see the custom check just because its on the build classpath, it needs to be added as an annotation processor, and I couldn't figure out how to do that.